pipeline/vars/common.groovy

```diff
@@ -37,7 +37,6 @@ def runTestSuite() {
     cmd += " --instances-name ${instanceName}"
 
     // Append test suite specific
-    cmd += " --inventory ${env.sutVMConf}"
```

Comment:
Please keep this statement. In case of not being set, we can use the default options.